### PR TITLE
Improve DisallowEqualOperators message when not equal operator is <>

### PR DESCRIPTION
**Pull request by carusogabriel**

### CODE REVIEW

File: SlevomatCodingStandard/Sniffs/ControlStructures/DisallowEqualOperatorsSniff.php [hunk 1]
```diff
@@ -36,7 +36,8 @@ public function process(\PHP_CodeSniffer\Files\File $phpcsFile, $operatorPointer
 				$phpcsFile->fixer->endChangeset();
 			}
 		} else {
-			$fix = $phpcsFile->addFixableError('Operator != is disallowed, use !== instead.', $operatorPointer, self::CODE_DISALLOWED_NOT_EQUAL_OPERATOR);
+			$differentOperator = $tokens[$operatorPointer]['content'];
+			$fix = $phpcsFile->addFixableError('Operator ' . $differentOperator . ' is disallowed, use !== instead.', $operatorPointer, self::CODE_DISALLOWED_NOT_EQUAL_OPERATOR);
```
Comment (greg0ire) [inline SlevomatCodingStandard/Sniffs/ControlStructures/DisallowEqualOperatorsSniff.php]: Small cs suggestion: fewer variables, shorter lines:
```php
<?php
$fix = $phpcsFile->addFixableError(sprintf(
    'Operator %s is disallowed, use !== instead.',
    $tokens[$operatorPointer]['content']
), $operatorPointer, self::CODE_DISALLOWED_NOT_EQUAL_OPERATOR);
```

Comment (kukulich) [reply SlevomatCodingStandard/Sniffs/ControlStructures/DisallowEqualOperatorsSniff.php]: Yes, sprintf is much better.

Comment (carusogabriel) [reply SlevomatCodingStandard/Sniffs/ControlStructures/DisallowEqualOperatorsSniff.php]: Done :+1:

**Comment by carusogabriel** [comment]
@kukulich Sure, gonna search how to test CS error messages and add :smile:

**Comment by kukulich** [comment]
Thanks.

**Pull request merged**